### DISCUSSION TITLE
OPA External Data Sizing Question

### DISCUSSION THREAD

**Discussion by Ronnie-personal** [open-policy-agent/community#372] [OPA and Rego] [answered]
Here there,
In OPA document https://www.openpolicyagent.org/docs/latest/external-data/#summary, it mentioned medium or large external data.
My questions are what size is considered as medium and what size is considered as large?

Thanks!

**Comment by anderseknert** [comment, marked as answer]
I don't know, but I'd say a few megabytes up to 50 could be considered medium, and anything above large.
